src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -80,6 +90,14 @@ def load_command_table(self, _):
         g.custom_command('get-versions', 'aks_get_versions',
                          table_transformer=aks_versions_table_format)
 
+    # AKS maintenance configuration commands
+    with self.command_group('aks maintenanceconfiguration', maintenance_configuration_sdk, client_factory=cf_maintenance_configurations) as g:
+        g.custom_command('list', 'aks_maintenanceconfiguration_list')
+        g.custom_show_command('show', 'aks_maintenanceconfiguration_show')
+        g.custom_command('add', 'aks_maintenanceconfiguration_add')
+        g.custom_command('update', 'aks_maintenanceconfiguration_update')
+        g.custom_command('delete', 'aks_maintenanceconfiguration_delete')
```

Comment:
Do we need to consider adding `confirmation=true` for the delete command?

Reply:
I think it's a good idea and I've gone ahead and added it :)

Reply:
Also, I would like to confirm if there is the same command `aks maintenanceconfiguration delete` in the CLI extension?

Reply:
Yes, this is the commands.py with this https://github.com/Azure/azure-cli-extensions/blob/97ed55b32bd1bc792eb85ddad5ee043167ae134e/src/aks-preview/azext_aks_preview/commands.py#L150C1-L155C74

Reply:
OK, so I think we need to keep consistent with CLI extension, otherwise, this may confuse or even block the customer's automated scripts